Pod/Classes/PINRemoteImageManager.h

```diff
@@ -184,6 +184,14 @@ typedef void(^PINRemoteImageManagerProgressDownload)(int64_t completedBytes, int
 - (nonnull id<PINRemoteImageCaching>)defaultImageCache;
 
 /**
+ * Set the custom header directly to every requests. Headers set from this method will override any header from NSURLSessionConfiguration.
```

Comment:
Could the first sentence say "Sets a custom header to be included in every request"